float32 ForwardMaxVelX 
float32 BackwardMaxVelX 
float32 MaxVelTheta 
float32 AccLimitX 
float32 AccLimitTheta 
float32 TurningRadius 
float32 Wheelbase 
float32 GoalToleranceXY 
float32 GoalToleranceYaw 
float32 MinObstacleDistance 
float32 ObstacleInflationRadius 
float32 DynamicObstacleInflationRadius 
float32 DTRef
float32 DTHysteresis 
bool IncludeDynamicObstacles
bool IncludeCostmapObstacles
bool OscillationRecovery
bool AllowInitializeWithBackwardMotion
bool SaveSettings